src/pages/Employer/WriteDocuments/ApplicantDocumentsDetailPage.tsx

```
@@ -34,54 +52,36 @@ const ApplicantDocumentsDetailPage = () => {
+                  type={property}
+                  reason={data?.data[property]?.reason}
+                  setIsLoading={setIsLoading}
+                  setModalContent={(content: SuccessModalContent) =>
+                    setSuccessModalContent(content)
+                  }
+                />
+              ) : (
+                <div className="w-full relative rounded-[1.125rem] flex flex-col items-start justify-start cursor-pointer">
+                  <InfoCardLayout
+                    icon={<YellowDocumentIcon />}
+                    title={EmployerDocumentTypeInfo[property].name}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)